extensions/wikia/AuthModal/css/AuthModal.scss

```diff
@@ -0,0 +1,72 @@
+@import 'skins/shared/mixins/transform';
+
+@keyframes auth-throbber-spin {
+	100% {
+		@include transform (rotate(360deg));
+	}
+}
+
+.auth-blackout {
+	background: rgba(0, 0, 0, .7);
+	box-sizing: border-box;
+	height: 100%;
+	left: 0;
+	padding: 50px 10%;
+	position: fixed;
+	top: 0;
+	width: 100%;
+	//overrides global navigation
+	z-index: 5000102;
+}
+
+.auth-modal {
```

Comment:
I don't know what UX Team thinks about it but blocking scrollbar could improve experience.